tests/test_scheduler.py

```diff
@@ -2190,3 +2176,61 @@ def test_scheduler_unqualified_imports(config):
 
     assert item.enable_imports
     assert item.children == ('other_routine',)
+
```

Comment:
I think it would be useful to add here also negative tests that verify behaviour and error messages when the transformation cannot be found or instantiation fails. This will likely be a reasonable common occurence, if only because of typos in the configuration file, faulty installations, or similar mistakes.